Audacia.Log.AspNetCore/ServiceCollectionExtensions.cs

```diff
@@ -88,18 +113,64 @@ public static IServiceCollection AddRequestBodyTelemetry(this IServiceCollection
         return services;
     }
 
+    /// <summary>
+    /// Configures logging of the http bodies for all http dependency requests.
+    /// </summary>
+    /// <exception cref="ArgumentNullException"><paramref name="services"/> is <see langword="null"/>.</exception>
+    /// <exception cref="ArgumentNullException"><paramref name="configuration"/> is <see langword="null"/>.</exception>
+    public static IServiceCollection ConfigureDependencyBodyContentLogging(
```

Comment:
I think a change to this method make sense with a new config type like a mentioned previously, currently this does the exact same as `ConfigureActionContentLogging`, even using the same config section name.

Reply:
Updated to use different config location.